api/src/main/java/org/openmrs/module/fgh/mpi/MpiHttpClient.java

```diff
@@ -178,8 +339,9 @@ private <T> T submitRequest(String requestPath, String data, Class<T> responseTy
 				out.close();
 			}
 			
-			if (connection.getResponseCode() != 200) {
+			if (!HTTP_REQUEST_SUCCESS_RANGE.contains(connection.getResponseCode())) {
```

Comment:
The documentation says that 2XX are success rest answer. In sante we have other response code (ex: 201 when the patient is successifuly created)

Reply:
It's good practice to only assume 200, at least OpenCR please only pass for 200 because the API can change  and they return other 2xx code with a meaning that needs to be treated otherwise

Reply:
Did you see this comment @daniel-chirinda ?

Reply:
You are saying that we have to assume only 200 for openCR?